_data/simple-icons.json

```diff
@@ -15440,6 +15440,11 @@
                 }
             }
         },
+        {
+            "title": "tldraw",
+            "hex": "FFFFFF",
```

Comment:
Not sure about `#FFFFFF`. The text color on their website is `#FAFAFA`, the footer color is `#CCCCCC`.

Reply:
oh yea thanks good catch, I just updated it to #FAFAFA